tests/resources/TestCaseBody.robot

```diff
@@ -459,3 +459,7 @@ Check Harbor Api Page
     ${Title}=  Get Title
     Should Be Equal  ${Title}  Harbor Swagger
     Retry Wait Element  xpath=//h2[contains(.,"Harbor API")]
+
+Prepare Imgpkg test files
```

Comment:
Naming as Imgpkg is not good enough, could you make it more readable?

Reply:
Has been modified to Prepare Image Package test files